### DISCUSSION TITLE
Unable to connect outside of network

### DISCUSSION THREAD

**Discussion by teeeebs** [thijsvanloef/palworld-server-docker#92] [Q&A] [answered]
I setup my server per the guides and i am able to connect to it if i use the local IP but if someone else tries to connect from outside the network, we get a connection timeout error. I setup port forwarding on my router and even added the server to DMZ to test if it was a possible networking issue but still not able to connect. Any help or suggestions are appreciated.

OS: Ubuntu server 22.04
Docker: Docker version 25.0.0

To install Docker, I followed the [official guide](https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository). After the install, i did the [postinstall ](https://docs.docker.com/engine/install/linux-postinstall/) stuff too.

**Comment by thijsvanloef** [comment]
Please make sure that the port forwarding works as intended with the correct protocol (UDP not TCP). If you are still having trouble, follow this guide: https://tice.tips/gaming/palworld-port-forward/

**Comment by teeeebs** [comment]
I followed that guide as close as possible but it's for Windows and I'm using Ubuntu server. I have port forwarding setup and verified with check-host.net that the result is "Open or filtered" but I still cant connect. I also temporarily disabled ufw on my server to make sure that wasnt causing issues.

To verify that port forwarding is working correctly, I setup nginx on my server and was able to access it from outside the network. Curious if anyone else was able to get this working with Ubuntu server.

**Reply by thijsvanloef** [reply, marked as answer]
As you mentioned in discord:

> I just figured it out. I have an asus router and there is an "Open NAT" section where you can add games. Looks like i had a game that had a port range that included 8211. The ports that are setup there arent shown in the port forwarding section of the router settings so thats why i never saw it until now. glad to have this figured out
